src/SmokeTests/TS001.java

```diff
@@ -0,0 +1,8 @@
+package src.SmokeTests;
+
+public class TS001 {
+    // TC001 for AC1 US008
+    // TC002 for AC1 US008
+    // TC003 for AC2 US008
```

Comment:
I need a full description please! :)

Reply:
I see, I can give you some classes you can understand the java code better

Reply:
TC003 is end-to-end test for purchase workflow

Reply:
just add one method!